src/server.js

```diff
@@ -131,7 +131,8 @@ const main = async () => {
 
   /**
    * @api {post} /api/event Log a user event
-   * @apiDescription Records the event to a log and forwards it to mixpanel. Optionally takes an authorization token which must be verified with Sam
+   * @apiDescription Records the event to a log and forwards it to mixpanel. Optionally takes an authorization token which must be verified with Sam.
+   *                 If properties['useBigQuery'] is true, only log the event (the logs will get sent to BigQuery via a log sink instead).
```

Comment:
Thanks for the feedback! I reworked the PR to use a `pushToMixpanel` property. I'm not sure who the intended consumer is of these logs actually, since I've only ever used mixPanel 🤔 I can update the README to record the new functionality!

Reply:
Awesome, thanks! The README helps :)

Reply:
Actually, sorry, can you update the docstring as well?  I'd also call out that `pushToMixpanel` defaults to true if unspecified.

Reply:
Ah thanks for catching that!